virttest/env_process.py

```diff
@@ -1240,6 +1240,22 @@ def preprocess(test, params, env):
             test.cancel("Got host qemu version:%s, which is not in %s" %
                         (host_qemu, required_qemu))
 
+    # Checking required virtio driver version, if not satisfied, cancel test
+    if params.get("required_virtio") and params.get("cdrom_virtio"):
+        required_virtio = params.get("required_virtio")
```

Comment:
Then we have to give a default range here to make sure it won't affect general tests.